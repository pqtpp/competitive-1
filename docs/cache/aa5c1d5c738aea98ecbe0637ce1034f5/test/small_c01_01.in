198 217
173 164 1
173 25 0
164 182 1
164 152 0
164 52 0
52 51 1
173 139 1
164 5 1
164 20 1
139 46 0
52 53 0
20 58 1
164 119 0
152 41 1
58 34 0
139 129 0
173 62 1
164 108 1
34 100 1
129 151 1
53 64 0
53 114 0
182 80 1
53 36 0
173 161 0
108 183 1
36 189 0
114 26 0
183 169 1
34 125 1
41 167 1
169 74 1
62 1 1
152 103 0
51 104 1
139 6 0
103 176 1
20 72 1
103 94 1
53 13 0
169 143 1
103 138 0
114 68 0
151 37 0
176 132 1
64 67 0
46 22 1
46 29 0
53 43 1
67 66 1
58 49 0
152 81 1
62 185 1
74 145 0
53 54 1
41 84 0
100 101 0
164 90 0
46 150 1
66 118 0
119 133 0
62 106 1
74 91 0
25 174 1
167 165 1
54 98 1
6 141 0
161 175 0
37 179 0
36 17 0
66 197 1
72 92 1
84 24 1
22 28 0
72 87 0
51 102 1
62 50 1
66 181 1
119 155 0
141 3 0
197 120 0
132 75 1
176 88 1
138 111 1
53 190 1
36 11 1
145 12 1
25 7 1
46 10 1
167 186 1
98 144 1
20 93 0
183 109 1
169 77 1
67 124 0
145 130 0
75 166 1
22 171 0
91 79 1
13 148 1
119 160 1
171 33 0
151 168 0
7 47 0
114 8 0
79 113 1
173 172 0
5 48 1
93 188 0
124 45 1
28 57 1
91 156 1
109 70 0
118 30 0
80 21 0
161 82 0
3 170 1
138 59 1
48 196 0
174 99 0
68 60 1
84 96 0
119 163 1
129 71 0
155 180 1
92 39 0
170 127 1
48 16 0
24 44 0
34 110 1
96 135 1
48 159 0
36 83 1
179 195 1
90 187 1
99 15 1
80 178 0
182 19 0
60 128 0
110 105 1
58 56 0
165 38 1
50 69 0
168 61 1
176 89 0
170 18 0
98 157 1
8 32 1
109 123 1
25 65 1
159 142 0
132 73 1
50 117 1
62 184 0
45 121 0
129 162 0
46 55 0
157 149 0
118 63 0
188 134 0
80 4 0
64 112 0
150 107 0
33 78 0
195 85 0
90 194 1
159 35 1
68 42 0
30 76 1
101 126 1
81 137 0
66 0 1
69 177 0
132 116 1
144 154 1
41 140 1
54 122 1
84 95 1
102 86 0
187 158 0
52 2 1
50 153 1
84 146 0
67 147 1
8 192 1
175 40 1
15 97 1
70 193 0
42 27 0
153 191 0
188 9 0
122 115 1
96 23 0
65 136 1
58 31 1
99 131 0
108 14 0
44 107 1
9 40 0
146 49 1
132 84 1
197 75 0
158 168 0
184 125 0
75 148 1
168 64 1
126 119 0
167 117 1
197 184 1
138 175 1
28 91 0
98 171 0
52 77 1
142 196 0
134 155 0
95 151 1
77 171 1
