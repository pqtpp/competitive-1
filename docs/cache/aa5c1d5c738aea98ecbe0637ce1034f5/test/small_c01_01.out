95
171 59 113 158 49 45 20 10 4 3 12 21 23 26 27 33 39 41 42 57 60 69 78 94 104 161 167 188 206 214 159 108 91 190 198 0 1 24 67 106 115 2 137 5 6 9 15 35 47 66 79 123 155 156 7 11 14 50 140 13 55 121 182 192 16 153 203 17 196 18 56 19 43 68 102 202 179 22 114 136 160 25 28 30 31 53 62 95 210 73 97 101 163 211 32 34 36 144 37 74 38 40 44 48 51 170 52 54 58 162 61 63 119 195 64 70 80 201 71 125 72 128 75 178 76 142 172 77 82 83 84 85 86 87 103 88 89 90 92 112 187 93 96 98 99 100 105 107 118 127 131 150 109 154 110 111 116 145 117 120 138 122 124 126 129 130 132 133 164 135 139 141 143 146 157 147 148 149 151 152 165 166 168 173 174 175 176 177 180 181 189 183 184 186 191 193 194
